src/ai/backend/common/clients/valkey_client/valkey_live/client.py

```diff
@@ -594,3 +594,13 @@ def _active_app_connection_value(
         :return: The value for active app connections.
         """
         return f"{kernel_id}:{service}:{stream_id}"
+
+    @valkey_live_resilience.apply()
+    async def exists(self, keys: Sequence[str]) -> int:
+        """
+        Check if keys exist.
+
+        :param keys: List of keys to check.
+        :return: Number of keys that exist.
+        """
+        return await self._client.client.exists(cast(list[str | bytes], list(keys)))
```

Comment:
From here, receive parameter as a list.

Reply:
Ah, It's for `str | bytes` type issue. That's not a problem. keep Sequence type.